pkg/curatedpackages/package.go

```diff
@@ -0,0 +1,38 @@
+package curatedpackages
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	api "github.com/aws/eks-anywhere-packages/api/v1alpha1"
```

Comment:
nit: I would name this `packagesv1`. It follows the pattern we have been trying to move towards when importing api struct packages.